src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java

```diff
@@ -88,14 +88,14 @@ public ResponseEntity<Resource> preview(Integer mediaId) {
     public ResponseEntity<byte[]> preview2(Integer mediaId, @RequestHeader(value = "Range", required = false) String httpRangeList) {
         InputStream audioMediaFile = audioService.preview(mediaId);
 
+        byte[] bytes = IOUtils.toByteArray(audioMediaFile);
         if (httpRangeList == null) {
-            byte[] bytes = IOUtils.toByteArray(audioMediaFile);
             return ResponseEntity.status(HttpStatus.OK)
-                .header("Content-Type", "audio/mpeg")
+                .header("Content-Type", "application/octet-stream")
+                .header("Content-Disposition", "attachment; filename=\"" + mediaId.toString() + ".mp3\"")
```

Comment:
I think this header will cause the browser to handle a file download - as this is what I'm using for the download transcript endpoint

Reply:
Yeah, I expect so, I'm just trying a different header to see if that's what's causing the request to be blocked.